frida_tools/repl.py

```diff
@@ -757,6 +760,92 @@ def _cancel_line(self):
             self._result = (None, KeyboardInterrupt())
             self._cond.notify()
 
+if os.environ.get("TERM", '') == "dumb":
```

Comment:
The convention is to use double-quoted strings for strings, and single-quoted for enum/magic values:

```suggestion
if os.environ.get("TERM", "") == 'dumb':
```

(In hindsight this is a bit nutty, but let's try to stick to it for now.)